src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,21 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      {post.user !== undefined && <UserInfo user={post.user} />}
```

Comment:
Your post.user will never be undefined as your `getUserById function` returns only `null` or `user Object`.
So in this case its better to write `{post.user && <UserInfo user={post.user} />}` or 
`{post.user !== null && <UserInfo user={post.user} />}`

Reply:
makes sense

In your case, you can simplify

```suggestion
      {post.user  && <UserInfo user={post.user} />}
```